### ISSUE TITLE
How to get all search results? Linked in shows only first 1000

### ISSUE THREAD

**Issue by maazt51** [tomquirk/linkedin-api#236]
I want know that in voyager api call the start is limited to 999 only. How to get pass this 999?

**Comment by lodye509** [comment]
is this a daily limit?

**Comment by redo-oss** [comment]
same problem .

**Comment by maazt51** [comment]
> is this a daily limit?

No, its about searching on the search bar. LinkedIn website will render only 10 searches per page and it shows max of 100 pages.

**Comment by maazt51** [comment]
> same problem .

What do you want to search in? like peoples? or companies?

**Comment by manuelrech** [comment]
i was trying to search for 1st connections, how can i manage to get more than 1000 results?
